src/Spd.Manager.Licence/BizLicAppValidation.cs

```diff
@@ -94,9 +94,6 @@ public BizLicAppSubmitRequestValidator(IConfiguration configuration)
         // General validations
         RuleFor(r => r.LatestApplicationId).NotEmpty();
         RuleFor(r => r.OriginalLicenceId).NotEmpty();
-        RuleFor(r => r.PreviousDocumentIds)
```

Comment:
do permit/swl has the same validator?
if yes, please remove them too.

Reply:
there is no rule for PreviousDocumentIds in permit and swl validations